keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/DeliveryConfigRepositoryTests.kt

```diff
@@ -483,7 +531,12 @@ abstract class DeliveryConfigRepositoryTests<T : DeliveryConfigRepository, R : R
 
           test("deletes related application pause records") {
             repository.deleteByApplication(deliveryConfig.application)
-            expectThat(pausedRepository.getPause(PauseScope.APPLICATION, deliveryConfig.application)).isNull()
+            expectThat(
```

Comment:
Just out of curiosity, do you have something configured in IntelliJ that's auto-applying formatting rules in the files you touch? I'd love to turn that on too.

Reply:
No I think I accidentally did the format command. You can set it to automatically apply on commit but I don't like to because it creates a big delta. I'd rather isolate formatting changes into their own commit. I'll back the unnecessary stuff out here.